react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -1,30 +1,16 @@
 import React from 'react';
-import styled from 'styled-components';
-import { createGlobalStyle } from 'styled-components';
 import CollaborationTool from './collaborationTool';
 
 import { Title, Title2, Title3, Title4 } from '../../StyleComponents/headings';
-
-const GlobalStyle = createGlobalStyle`
```

Comment:
So we'll just put them in file I'm guessing?

Reply:
yes exactly